crm_phonecall/views/crm_phonecall_view.xml

```diff
@@ -100,6 +100,11 @@
                         <field name="priority" widget="priority"/>
                         <field name="opportunity_id" context="{'opportunity_id': opportunity_id}"/>
                     </group>
+                    <group string="Tracking" groups="base.group_multi_company,base.group_no_one" name="categorization">
+                        <field name="campaign_id"/>
+                        <field name="source_id"/>
+                        <field name="medium_id"/>
```

Comment:
Shouldn't this fields also be available to record at the logged calls editable tree?

Reply:
Mmm I thought about that, I'm not sure. Some users could consider that excessive information, others necessary... @pedrobaeza @rafaelbn help here please.

Reply:
Isn't there any specific security group for this?